Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerHybridInformation.ps1

```diff
@@ -150,5 +150,122 @@ Function Invoke-AnalyzerHybridInformation {
+                            -DisplayCustomTabNumber 2
+                    } elseif ($connector.CertificateMatchDetected -eq $false) {
+                        $AnalyzeResults | Add-AnalyzedResultInformation -Details "The configured Tls Certificate was not found on the server. This may cause mail flow issues." `
+                            -DisplayGroupingKey $keyHybridInformation `
+                            -DisplayWriteType $cloudConnectorWriteType `
+                            -DisplayCustomTabNumber 2
+                    } else {
+                        $AnalyzeResults | Add-AnalyzedResultInformation -Name "Certificate Thumbprint(s)" `
+                            -DisplayGroupingKey $keyHybridInformation
+
+                        foreach ($thumbprint in $($connector.CertificateInformation).keys) {
+                            $AnalyzeResults | Add-AnalyzedResultInformation -Details $thumbprint `
+                                -DisplayGroupingKey $keyHybridInformation `
+                                -DisplayCustomTabNumber 2
+                        }
+
+                        $AnalyzeResults | Add-AnalyzedResultInformation -Name "Lifetime In Days" `
+                            -DisplayGroupingKey $keyHybridInformation
+
+                        foreach ($thumbprint in $($connector.CertificateInformation).keys) {
+                            switch ($($connector.CertificateInformation)[$thumbprint]) {
+                                { $_ -ge 60 } { $certificateLifetimeWriteType = "Green"; break }
+                                { $_ -ge 30 } { $certificateLifetimeWriteType = "Yellow"; break }
+                                Default { $certificateLifetimeWriteType = "Red" }
+                            }
+
+                            $AnalyzeResults | Add-AnalyzedResultInformation -Details ($connector.CertificateInformation)[$thumbprint] `
+                                -DisplayGroupingKey $keyHybridInformation `
+                                -DisplayWriteType $certificateLifetimeWriteType `
+                                -DisplayCustomTabNumber 2
+                        }
+                    }
+
+                    $connectorCertificateMatchesHybridCertificateWritingType = "Yellow"
+                    if (($connector.TlsCertificateSet) -and
+                        (-not([System.String]::IsNullOrEmpty($exchangeInformation.GetHybridConfiguration.TlsCertificateName))) -and
+                        ($connector.TlsCertificateName -eq $exchangeInformation.GetHybridConfiguration.TlsCertificateName)) {
+                        $connectorCertificateMatchesHybridCertificate = $true
+                        $connectorCertificateMatchesHybridCertificateWritingType = "Green"
+                    }
+
+                    $AnalyzeResults | Add-AnalyzedResultInformation -Name "Certificate Matches Hybrid Certificate" -Details $connectorCertificateMatchesHybridCertificate `
```

Comment:
`$connectorCertificateMatchesHybridCertificate` should be set as `$false` to have something come up. Otherwise by default it will be blank.

Reply:
You're right. Fixed.